web/packages/extension/src/background.ts

```diff
@@ -9,6 +9,75 @@ async function contentScriptRegistered() {
 }
 
 async function enable() {
+    if (utils.declarativeNetRequest) {
+        const playerPage = utils.runtime.getURL("/player.html");
+        const rules = [
+            {
+                id: 1,
+                action: {
+                    type: chrome.declarativeNetRequest.RuleActionType.REDIRECT,
```

Comment:
In `chrome`, this is part of an `enum`: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/chrome/index.d.ts#L12198. I can't use utils.declarativeNetRequest.RuleActionType.REDIRECT because in `browser` it's a `type` instead: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/webextension-polyfill/namespaces/declarativeNetRequest.d.ts#L413